### DISCUSSION TITLE
Vote for Project Topic

### DISCUSSION THREAD

**Discussion by yehandesilva** [yehandesilva/monkeyPoll#2] [Polls]
Vote for which project topic you would like to do. I've made the CI integration on Azure independent of the project topic, but I've already titled this repository monkeyPoll (Topic or name can change).

**Comment by yehandesilva** [comment]
I think the mini monkey survey will be a good project. A simple form application that just needs to store and retrieve surveys and questions. The only tricky part will be the closing of surveys.

The others are not bad as well. I've talked a look at the jaccard algorithm and it's not complicated.

**Reply by pathumd** [reply]
Agreed, I am on board for going with this project as well.

**Reply by MichaelKyrollos** [reply]
Good idea, seems we are all on the same page

**Reply by AminZeina** [reply]
Agreed!
